doc/developer-guides/hld/hld-devicemodel.rst

```diff
@@ -89,8 +77,10 @@ options:
        --part_info: guest partition info file path
        --enable_trusty: enable trusty for guest
        --ptdev_no_reset: disable reset check for ptdev
-       --intr_monitor: enable interrupt storm monitor, params:
-       		threshold/s,probe-period(s),delay_time(ms),delay_duration(ms)
+       --debugexit: enable debug exit function
+       --intr_monitor: enable interrupt storm monitor
+       --vtpm2: Virtual TPM2 args: sock_path=$PATH_OF_SWTPM_SOCKET
+            its params: threshold/s,probe-period(s),delay_time(ms),delay_duration(ms)
```

Comment:
Are these parameters (`thresholds, probe-period, etc`) applicable to `--vtpm2` or should it be moved up one line, under `--intr_monitor`? I think aae70db6b97e92bedba6fbc3dba17a80ef400806 actually introduced that issue in the help message in the code, if confirmed, can you fix this document *and* also include the same fix in the code (`devicemodel/core/main.c`)?